src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.NodeMapBuilder.cs

```diff
@@ -85,15 +85,15 @@ public static void AddToMap(BoundNode root, Dictionary<SyntaxNode, ImmutableArra
                                     Debug.Assert(
                                         ((BoundTypeExpression)existing[i]).Type == ((BoundTypeOrValueExpression)added[i]).Type,
                                         string.Format(
-                                            CultureInfo.InvariantCulture,
+                                            System.Globalization.CultureInfo.InvariantCulture,
```

Comment:
What prompted this change?

Reply:
Uncommenting out this code and finding that it doesn't compile.

---
In reply to: [167697572](https://github.com/dotnet/roslyn/pull/24756#discussion_r167697572) [](ancestors = 167697572)